### ISSUE TITLE
[Windows Terminal – Settings – Startup]: Voice Access does not display numbers for the collapsed 'Launch size' and 'Launch parameters' controls.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#20604]
### Windows Terminal version

1.25.1912.0

### Windows build number

28120.2738

### Other Software

**Test Environment:**
OS Version: Windows 11 version 26H1 (OS Build 28120.2738)
App Name: Windows Terminal Preview

### Steps to reproduce

**Steps for turning on Voice access:**
Settings>Accessibility>Speech>Voice Access.

**Repro Steps:**
1.	Launch the 'Windows Terminal' application.
2.	Press 'Ctrl + ,' keys to open 'Settings'.
3.	Navigate to Settings > Startup.
4.	Turn on Voice Access.
5.	Observe the 'Launch size' and 'Launch parameters' controls.
6.	Observe the Voice Access numbers displayed for these controls.

**Observation:**
The issue reproduces throughout the application with the same controls.

**User Experience:**
Voice Access users may be unable to directly identify or interact with the collapsed controls using their assigned Voice Access numbers, making navigation and interaction less efficient.

**WCAG Reference Link:**
https://aka.ms/MAS4.3.1

**Attachment:**
https://onedrive.cloud.microsoft/:f:/a@rc32u9qu/S/IgDoKee8JijBTJx4JetP9je_AX0bBhU7gKJ3Y51Tn3rhwX0?e=yq6Sw4

### Expected Behavior

Voice Access should display a unique number for the 'Launch size' and 'Launch parameters' controls so that users can identify and interact with them using Voice Access commands.

### Actual Behavior

When the 'Launch size' and 'Launch parameters' controls are collapsed, Voice Access does not display a number for these controls.
